docker/compose-local-dev.overrides.yml

```diff
@@ -36,19 +36,7 @@ services:
       - "5005:5005"
     volumes:
       - ../tmp/local/:/data/
-
-  sequencer:
-    environment:
-      JAVA_OPTS: -XX:+UnlockExperimentalVMOptions -XX:-UseG1GC -XX:+UseZGC
-
-  zkbesu-shomei:
-    environment:
-      JAVA_OPTS: -XX:+UnlockExperimentalVMOptions -XX:-UseG1GC -XX:+UseZGC
-
-  l1-el-node:
-    environment:
-      JAVA_OPTS: -XX:+UnlockExperimentalVMOptions -XX:-UseG1GC -XX:+UseZGC
   
   l2-node-besu:
     environment:
-      JAVA_OPTS: -XX:+UnlockExperimentalVMOptions -XX:-UseG1GC -XX:+UseZGC
+      JAVA_OPTS: -XX:+UnlockExperimentalVMOptions -XX:-UseG1GC -XX:+UseZGC
```

Comment:
@Filter94 I think we might need to keep the above overrides for `l2-node-besu` for now given that the pipeline was failing at the e2e geth test and I see local e2e geth test would still fail with OOM for max heap size set as `-Xmx2g`, provided that e2e geth test would be removed in the future, maybe we could keep this overrides for now, and in parallel, I'd search for a less heap size consumption alternative for transaction exclusion e2e test